README.md

```diff
@@ -55,6 +55,21 @@ threads to use during execution. For additional options please see:
 python tests/eigenwave3d.py --help
 ```
 
+##### Profiling
+
+If the PAPI library is found on your system during the initial build,
+Opesci-FD can also provide profiling information, such as the achieved
+number of MFlops/s during automated runs. To enable this feature
+simply add the `--profiling` flag to the example command above. The
+user can also supply a list of PAPI event names, for example
+`PAPI_TOT_CYC` or `PAPI_FP_OPS`, via the `--papi-events` flag:
+```
+python tests/eigenwave3d.py -c gcc -x --n 4 --profiling --papi-events PAPI_TOT_CYC PAPI_FP_OPS
```

Comment:
Yesterday, I tried copy and past this example line, but I got an error. Wouldn't it be more convenient change the 'gcc' for 'g++' to give users higher chance of success on their first attempt?